bot/kodiak/entrypoints/worker.py

```diff
@@ -75,6 +123,14 @@ async def main() -> NoReturn:
                 queue.start_webhook_worker(queue_name=task_meta.queue_name)
             else:
                 assert_never(task_meta.kind)
+        if ingest_queue_watcher.done():
+            logger.info("worker task failed", kind="ingest_queue_watcher")
+            exception = ingest_queue_watcher.exception()
```

Comment:
shouldn't we check if this exception is non `None` before logging / sending off to sentry?

Reply:
it shouldn't ever finish